src/Latte/Engine.php

```diff
@@ -149,6 +149,27 @@ public function compile($name)
 
 
 	/**
+	 * Puts specific latte to cache
+	 * @param string $name
+	 *
+	 * @throws \Latte\RuntimeException
+	 */
+	public function warmupCache($name)
+	{
+		if (!isset($this->tempDirectory)) {
+			throw new RuntimeException('Temp directory is not set, so putting to cache is useless.');
+		}
+
+		$this->setAutoRefresh(FALSE);
```

Comment:
I don't think so. If you are creating this, you do it in clean instance, with another directory, or not?